### PR TITLE
LPS-139939 Deploy AWS translator by default

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:forward

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
ci:test:relevant

File: modules/apps/translation/translation-translator-aws/src/main/java/com/liferay/translation/translator/aws/internal/translator/AWSTranslator.java [hunk 1]
```diff
@@ -101,15 +102,25 @@ protected void activate(Map<String, Object> properties) {
...
+				StaticCredentialsProvider.create(
+					AwsBasicCredentials.create(
+						_awsTranslatorConfiguration.accessKey(),
+						_awsTranslatorConfiguration.secretKey()))
+			).region(
+				Region.of(_awsTranslatorConfiguration.region())
+			).build();
+		}
+		else {
+			_translateClient = null;
```
Comment (AliciaGarciaGarcia) [inline modules/apps/translation/translation-translator-aws/src/main/java/com/liferay/translation/translator/aws/internal/translator/AWSTranslator.java:117]: We are assigning `_translateClient` to null value. 
There is some way that can we try to use it being null? (on the line 129) (like being enabled and after disabling) or it's a very extreme corner case.
it's worth a NPE check?

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**